pkg/e2/v1beta1/node.go

```diff
@@ -168,17 +170,23 @@ func (n *e2Node) Control(ctx context.Context, message *e2api.ControlMessage) (*e
 	return &response.Outcome, nil
 }
 
-func (n *e2Node) Subscribe(ctx context.Context, name string, sub e2api.SubscriptionSpec, indCh chan<- e2api.Indication) (e2api.ChannelID, error) {
+func (n *e2Node) Subscribe(ctx context.Context, name string, sub e2api.SubscriptionSpec, indCh chan<- e2api.Indication, opts ...Option) (e2api.ChannelID, error) {
```

Comment:
A well-designed API should not allow users to do the wrong thing. This essentially allows node-level options to be added to the `Subscribe` call. We should instead add a separate `SubscribeOption` interface that’s only implemented by subscribe options.